pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -2016,6 +2096,22 @@ let
         };
       };
 
+      hediet.vscode-drawio = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "vscode-drawio";
+          publisher = "hediet";
+          version = "1.6.6";
+          sha256 = "sha256-SPcSnS7LnRL5gdiJIVsFaN7eccrUHSj9uQYIQZllm0M=";
+        };
+        meta = {
+          description = "This unofficial extension integrates Draw.io into VS Code.";
+          downloadPage = "https://marketplace.visualstudio.com/items?itemName=hediet.vscode-drawio";
+          homepage = "https://github.com/hediet/vscode-drawio";
+          license = lib.licenses.gpl3;
```

Comment:
Missed this before, but gpl3 is deprecated and either gpl3Plus or gpl3Only should be used instead.

Reply:
are you sure? It is used in the project 
https://github.com/hediet/vscode-drawio/blob/master/LICENSE.md

Reply:
@jopejoe1 Can you check the licenses now, are they all right?

Reply:
Yeah, they look good now